### PR TITLE
Remove obsolete ops

### PR DESCRIPTION
**Pull request by Ndpnt**
Ops is extracted into the [Open Terms Archive deployment Ansible collection](https://github.com/OpenTermsArchive/deployment)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,8 +90,8 @@ For command-line examples and documentation, we follow the [docopt usage pattern
 - mutually exclusive elements are given between `(` and `)` and separated by `|`.
 
 ```diff
-- npm start [ $service_id ] [, $service_id, ...]
-+ npm start [service_id]...
+- npm start --services [ $service_id ] [, $service_id, ...]
++ npm start --services [service_id...]
```
Comment (MattiSG) [inline CONTRIBUTING.md:94]: This would not work. We would need:
```suggestion
- npm start -- --services [ $service_id ] [, $service_id, ...]
+ npm start -- --services [service_id...]
```

For the purposes of illustrating docopt, I am not sure this change is very relevant, I think it makes things more complicated. I suggest a rollback:
```suggestion
- npm start [ $service_id ] [, $service_id, ...]
+ npm start [service_id...]
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -100,8 +100,8 @@ In order to improve the understandability of commands, we document all CLI optio
 
 
 ```diff
-- ansible-playbook -i production.yml app.yml -l france -t stop
-+ ansible-playbook --inventory production.yml app.yml --limit france --tag stop
+- npm start -s [$service_id] -r
++ npm start --services [$service_id] --refilter-only
```
Comment (MattiSG) [inline CONTRIBUTING.md:104]: Same as above, this would not work and is more confusing for the pursued goal:

```suggestion
- ota track -s [$service_id] -r
+ ota track --services [$service_id] --refilter-only
```

File: config/production.json [hunk 3]
```diff
@@ -1,7 +1,4 @@
 {
-  "services": {
-    "declarationsPath": "../declarations/declarations"
```
Comment (MattiSG) [inline config/production.json:3]: Why is this changed? I'm worried of consequences on production instances.

**Review by MattiSG — CHANGES_REQUESTED** [review]

**Comment by MattiSG** [comment]
Nice, I'm excited about starting to get the benefits of decreased complexity thanks to our investments in improving the architecture 👍

**Review by MattiSG — APPROVED** [review]
The changes I suggested are to me relevant but are not blocking.

File: config/production.json (hunk 3 above)
Comment (Ndpnt) [reply config/production.json:3]: It's obsolete and overridden by instances: https://github.com/OpenTermsArchive/contrib-declarations/pull/748/files#diff-07e0c2ac42cdc22639d12b2282f02825cda2dcf94c741cad1f57706e93b62d2fR5

**Review by martinratinaud — APPROVED** [review]
I pushed an additional commit to link to the deployment repo from the README.

Apart from that, looks good
